platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -1044,6 +1044,10 @@ public String[] getGrantedPermissions() {
 		return PermissionsUtil.getGrantedPermissions(getActivity());
 	}
 
+	public String getCACertificates() {
+		return GodotNetUtils.getCACertificates();
+	}
```

Comment:
This is only intended for use by the native code, so you can make the method `private` and add the `@Keep` annotation to denote it should not be stripped by proguard.